.github/workflows/test.yml

```diff
@@ -0,0 +1,147 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Run hstreamdb operator test case
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  UnitTest:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        use-existing-cluster:
+          - true
+          - false
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19.3
+
+    - name: Go Build Cache
+      uses: actions/cache@v3
+      with:
+        path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+
+    - name: Install kubebuilder
+      run: |
+        OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+        ARCH=$(uname -m | sed 's/x86_64/amd64/')
+        curl -fsL "https://storage.googleapis.com/kubebuilder-tools/kubebuilder-tools-1.16.4-${OS}-${ARCH}.tar.gz" -o kubebuilder-tools
+        tar -zxvf kubebuilder-tools
+        sudo mv kubebuilder/ /usr/local/kubebuilder
+
+    - name: Mod tidy and install ginkgo
+      run: |
+        go mod tidy
+        go get github.com/onsi/ginkgo/v2/ginkgo/generators@v2.5.0
+        go get github.com/onsi/ginkgo/v2/ginkgo/internal@v2.5.0
+        go get github.com/onsi/ginkgo/v2/ginkgo/labels@v2.5.0
+        go install github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
Do we need `go get` in here ? I think we just need `go install`

Reply:
I have tried `go install`, but it told me to run `go get` to download the repositories above

Reply:
I also encounter this problem in my dev environment

Reply:
I can direct run `ginkgo ./...` in my local, did you miss some thing ?